presto-benchmark-driver/src/main/java/io/prestosql/benchmark/driver/BenchmarkQueryRunner.java

```diff
@@ -261,12 +262,12 @@ public void handleFailure(Exception e)
         }
     }
 
-    private long getTotalCpuTime()
+    private long getTotalCpuTime(String user)
     {
         long totalCpuTime = 0;
         for (URI server : nodes) {
             URI addressUri = uriBuilderFrom(server).replacePath("/v1/jmx/mbean/java.lang:type=OperatingSystem/ProcessCpuTime").build();
-            String data = httpClient.execute(prepareGet().setUri(addressUri).build(), createStringResponseHandler()).getBody();
+            String data = httpClient.execute(prepareGet().setHeader(PrestoHeaders.PRESTO_USER, user).setUri(addressUri).build(), createStringResponseHandler()).getBody();
```

Comment:
this might be easier to read with multi-line formatting, and extracting out request preparation.

```
            Request request = prepareGet()
                    .setHeader(PrestoHeaders.PRESTO_USER, user)
                    .setUri(uriBuilderFrom(server)
                            .replacePath("/v1/jmx/mbean/java.lang:type=OperatingSystem/ProcessCpuTime")
                            .build())
                    .build();
            String data = httpClient.execute(request, createStringResponseHandler()).getBody();
```